Update workaround for unnecessary live migrations (#1058)
harvester/docs: docs/upgrade/v1-5-x-to-v1-6-x.md

#### Before the upgrade

> 1. Edit the `harvester` ManagedChart and set `workloadUpdateMethods` to an empty list under `spec.values.kubevirt.workloadUpdateStrategy`:

Author: fixed.

   ```bash
   kubectl edit managedchart.management.cattle.io harvester -n fleet-local
   ```

   ```yaml
   spec:
     values:
      kubevirt:
        spec:

>           ...

Author: fixed

          workloadUpdateStrategy:
            workloadUpdateMethods: []
   ```

> 1. Verify that the kubevirt CR reflects the change:

Author: will keep the lowe case

#### Before the upgrade

> 1. Edit the `harvester` ManagedChart and set `workloadUpdateMethods` to an empty list under `spec.values.kubevirt.spec.workloadUpdateStrategy`:

Reviewer: ```suggestion
1. Edit the `harvester` ManagedChart resource. Under `spec.values.kubevirt.spec.workloadUpdateStrategy`, set the `workloadUpdateMethods` field to an empty list `[]`.
```

   ```bash
   kubectl edit managedchart.management.cattle.io harvester -n fleet-local
   ```

   ```yaml
   spec:
     values:
       kubevirt:
         spec:
           monitorAccount: rancher-monitoring-operator
           monitorNamespace: cattle-monitoring-system
           workloadUpdateStrategy:
             workloadUpdateMethods: []
   ```

   :::note

   Only change the `workloadUpdateMethods` value. Do not delete any other existing fields in the `harvester` ManagedChart.

   Note that the YAML snippet above is a partial excerpt, not a full resource manifest.

>    :::

Reviewer: ```suggestion
  :::caution

  Only modify the `workloadUpdateMethods` field. Do not modify or delete any other existing fields in the `harvester` ManagedChart resource.

  The YAML snippet is an excerpt, not a full resource manifest.

  :::
```

> 1. Verify that the kubevirt CR reflects the change:

Reviewer: ```suggestion
1. Verify that the KubeVirt CR reflects the change.
```

After the upgrade completes successfully, restore the `LiveMigrate` workload update method to re-enable CPU and memory hot-plugging.

> 1. Edit the `harvester` ManagedChart and restore `workloadUpdateMethods` under `spec.values.kubevirt.workloadUpdateStrategy`:

Author: fixed

   ```bash
   kubectl edit managedchart.management.cattle.io harvester -n fleet-local
   ```

   ```yaml
   spec:
     values:
       kubevirt:
         spec:
          ...
          workloadUpdateStrategy:
            workloadUpdateMethods:

>             - LiveMigrate

Author: fixed

After the upgrade completes successfully, restore the `LiveMigrate` workload update method to re-enable CPU and memory hot-plugging.

> 1. Edit the `harvester` ManagedChart to remove the `workloadUpdateStrategy` field under `spec.values.kubevirt.spec`:

Reviewer: ```suggestion
1. Edit the `harvester` ManagedChart resource. Under `spec.values.kubevirt.spec`, remove the `workloadUpdateStrategy` field.
```

   ```bash
   kubectl edit managedchart.management.cattle.io harvester -n fleet-local
   ```

   ```yaml
   spec:
     values:
       kubevirt:
         spec:
           monitorAccount: rancher-monitoring-operator
           monitorNamespace: cattle-monitoring-system
           # ... keep all other existing fields unchanged
   ```

> 1. Verify that the `LiveMigrate` method has been restored in the kubevirt CR:

Reviewer: ```suggestion
1. In the KubeVirt CR, verify that `LiveMigrate` has been restored in the `workloadUpdateMethods` field.
```
